packages/asc-core/src/components/ContextMenu/ContextMenuItem.ts

```diff
@@ -0,0 +1,31 @@
+import styled from '../../styled-components'
+import { ContextMenu } from './types'
+import Icon from '../Icon'
+import { getColor, getTypography } from '../../utils/themeUtils'
+
+const ContextMenuItem = styled.li<ContextMenu.ContextMenuItemProps>`
+  padding: 0;
+  margin: 0;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  height: 36px;
```

Comment:
The [design system mentions](https://zeroheight.com/7awj1hc9f/p/4701a1) using `34px` instead of `36px`